test/cloudflare_protected_email_address_test.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative './test_helper'
+require_relative '../lib/cloudflare_protected_email_address'
+
+describe CloudflareProtectedEmailAddress do
+  let(:email) do
+    CloudflareProtectedEmailAddress.new(obfuscated: 'b3d2dfd7dcc5d6c1d2f3d7dac3c6c7d2d7dcc09dd4dcc59dc3ca')
+                                   .human_readable
+  end
+
+  it 'will make an obfuscated email address human readable' do
+    email.must_equal 'aldovera@diputados.gov.py'
```

Comment:
it would be good to have another test here as well. I could make this test suite pass with a MUCH simpler implementation :D